Gems/PhysX/Code/Source/Pipeline/HeightFieldAssetHandler.cpp

```diff
@@ -126,20 +126,8 @@ namespace PhysX::Pipeline
         {
             if (header.m_assetDataSize > 0)
             {
-                // Version 1 doesn't have min/max heights, so only read this data for versions 2+.
-                if (header.m_assetVersion >= 2)
-                {
-                    readerStream.read(&physXHeightFieldAsset->m_minHeight, sizeof(float));
-                    readerStream.read(&physXHeightFieldAsset->m_maxHeight, sizeof(float));
-                }
-                else
-                {
-                    // In versions 0 & 1, the data is cooked assuming the data starts at origin (min height = 0)
-                    // and has a max height of 1024.0f.
-                    const float v1HardCodedMaxHeight = 1024.0f;
-                    physXHeightFieldAsset->m_minHeight = 0.0f;
-                    physXHeightFieldAsset->m_maxHeight = v1HardCodedMaxHeight;
-                }
+                readerStream.read(&physXHeightFieldAsset->m_minHeight, sizeof(float));
+                readerStream.read(&physXHeightFieldAsset->m_maxHeight, sizeof(float));
```

Comment:
Just curious, is there any particular value in storing these? I'm not opposed to it, but I don't think they're used anywhere?